lms/templates/courseware/accordion.html

```diff
@@ -80,3 +81,10 @@
 % for chapter in toc:
     ${make_chapter(chapter)}
 % endfor
+
+
+% if toc:
+    <%static:require_module module_name="js/courseware/accordion_events" class_name="AccordionEvents">
+        AccordionEvents();
+    </%static:require_module>
+% endif
```

Comment:
End all files with a newline.  You can probably configure your text editor / IDE to always do this.